src/lib/validators/openApiDiff.ts

```diff
@@ -247,16 +247,17 @@ export class OpenApiDiff {
     if (stderr) {
       throw new Error(stderr)
     }
-    const resolveSwagger = new ResolveSwagger(outputFilePath)
+
```

Comment:
Change is just moving this code earlier, and passing `map` to `new ResolveSwagger()` so it can include original locations in errors.